charts/tempo-distributed/values.yaml

```diff
@@ -465,6 +465,8 @@ distributor:
     loadBalancerIP: ''
     # -- If type is LoadBalancer limit incoming traffic from IPs.
     loadBalancerSourceRanges: []
+    # -- If type is LoadBalancer you can set it to 'Local' [preserve the client source IP](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)
+    externalTrafficPolicy: null
```

Comment:
```suggestion
    externalTrafficPolicy: {}
```

Reply:
Thanks for the review! I'm not sure about this one the field is a string and not an object so I applied what seemed to be a policy to me. 
![image](https://github.com/grafana/helm-charts/assets/10258608/b38b659f-2847-4574-8fab-77b9ad8dd514)

Reply:
You want  `externalTrafficPolicy` to be configurable right ?

Reply:
Yes exactly, or if they want to put nothing, it just relies on the default kubernetes value. Field not set

Reply:
> if they want to put nothing,

That's what we are doing when we set  `externalTrafficPolicy: {}` it is an empty object and user configurable.

Reply:
Having the default value to ``nil`` or ``{}`` will have the same effect. To not pass inside my ``with``. 
The thing is I noted that all the other default of strings are ususally ``nil`` and default of objects are usually ``{}``. I'm just following it for consistency.

Reply:
> Having the default value to `nil` or `{}` will have the same effect.

That's right!!